### ISSUE TITLE
CPU conflict check

### ISSUE THREAD

**Issue by ParaplegicRacehorse** [ParaplegicRacehorse/RainbowMiner#3]
https://github.com/ParaplegicRacehorse/RainbowMiner/blob/docs/Docs/rainbowminer.first-run.md#enable-conflict-check-if-running-cpu-hungry-gpu-miners-for-weak-cpus-yesno-defaultno

**Comment by RainbowMiner** [comment]
Yes, if this conflict management is enabled and a CPU hungry GPU algorithm (e.g. like Cuckaroo29d) is to be selected, RainbowMiner pre-calculates, if it is better to run the next best non-CPU hungry GPU algorithm, together with a CPU miner, or just disable CPU and start the CPU hungry GPU miner.
